setup.py

```diff
@@ -36,6 +36,7 @@
         "pydantic==1.10.8",
         "replicate==0.9.0",
         "duckduckgo-search==3.8.4",
+        "elasticsearch>=8.0.0",
```

Comment:
why is this a lower version?

Reply:
yes, i think latest version is 8.9.0.
Any specific reason?

Reply:
update: now using elasticsearch(8.9.0) as optional dependency as @cachho [suggested](https://github.com/embedchain/embedchain/pull/402#discussion_r1288843534).